.github/workflows/pullrequest.yml

```
@@ -306,6 +339,47 @@
         with:
           clean: false
 
+  try_install:
+    needs:
+      - prepare
+    runs-on: arc-shared
+    timeout-minutes: 10
+    # Ugly! But I want to get rate limit errors. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)